### PR TITLE
Add @steffilazerte to team page

### PR DESCRIPTION
**Pull request by maelle**
@stefaniebutland Not sure how we should proceed

* Either I erase the JSON edit after you've looked at [the preview](https://deploy-preview-620--ropensci.netlify.com/about/#team) and merge the PR with just the badge creation.

* Or you update this branch with more "adding intern to website" stuff.

### CODE REVIEW

File: themes/ropensci/static/css/sections.pcss [hunk 1]
```diff
@@ -138,6 +138,12 @@ article ol {
                 content: 'staff';
             }
         }
+        &.intern {
```
Comment (maelle) [inline themes/ropensci/static/css/sections.pcss]: these are the lines I explicitely added.

File: themes/ropensci/static/css/style-new.css [hunk 2]
```diff
@@ -689,6 +689,10 @@ q:before, q:after {
   background-color: #748290;
 }.team .label.staff:after{
   content: 'staff';
+}.team .label.intern{
```
Comment (maelle) [inline themes/ropensci/static/css/style-new.css]: these lines were changed after I compiled CSS cf https://github.com/ropensci/roweb2/blob/master/developer-notes.md#compiling-css

**Comment by stefaniebutland** [comment]
Thank you!! A bonus that you already added her picture and saved me that step.

Would you mind moving her to after Mark P and then merge?
(I'm not sure how to do that in this PR)

**Comment by stefaniebutland** [comment]
Ah. Yes please, with the GitHub avatar, and a hundred thank you's

**Pull request merged**